tests/acceptance/features/coreApiWebdavProperties/createFileFolder.feature

```diff
@@ -264,3 +264,12 @@ Feature: create files and folder
       | spaces           | /..fo       |
       | spaces           | /fo.xyz     |
       | spaces           | /fo.exe     |
+
+  @issue-10809
+  Scenario: create a folder with Transfer-Encoding: chunked header
+    Given using spaces DAV path
+    When user "Alice" creates folder "myFolder" with the following headers using the WebDAV API
+      | header            | value   |
+      | Transfer-Encoding | chunked |
```

Comment:
should we add something like this? so that we know why this test can fail in some environments.
```suggestion
    When user "Alice" creates folder "myFolder" with the following headers using the WebDAV API
      | header            | value   |
      # NOTE: requires curl >= 8.0
      | Transfer-Encoding | chunked |
```